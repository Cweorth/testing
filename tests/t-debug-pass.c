#include <cut.h>

TEST(debugSuccess) {
    DEBUG_MSG("Line: %d", __LINE__);
}
