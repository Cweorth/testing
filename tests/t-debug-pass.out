[  1] debugSuccess...........................................................[1;32mOK[0m
    debug messages:
      Line: 4 (debug-pass.c:4)


Summary:
  tests:       1
  succeeded:   1
  skipped:     0
  failed:      0
